### PR TITLE
HTML movie seat booking

### PR DESCRIPTION
**Pull request by vakulinina**
### HTML movie seat booking

[Demo](https://vakulinina.github.io/kottans-movie-seat-booking/)
[Code base](https://github.com/vakulinina/kottans-movie-seat-booking)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/vakulinina/html-movie-seat-booking/index.html [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@200;400;600;700&display=swap" rel="stylesheet">
+  <title>Book your seat</title>
+</head>
+
+<body>
+  <main>
+    <header>
+      <h1>Buy tickets</h1>
```
Comment (A-Ostrovnyy) [inline submissions/vakulinina/html-movie-seat-booking/index.html:16]: `h1` better use for the movie title.

File: submissions/vakulinina/html-movie-seat-booking/index.html [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+    </header>
+    <article class="movie">
+      <div class="movie-poster"></div>
+      <div class="movie-description">
+        <h2>Alita: Battle Angel</h2>
```
Comment (A-Ostrovnyy) [inline submissions/vakulinina/html-movie-seat-booking/index.html:21]: 🔥 nice movie choice).

File: submissions/vakulinina/html-movie-seat-booking/index.html [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+        <p>2019 &bull; Action | Sci-Fi</p>
+        <p><b>Synopsys</b></p>
```
Comment (A-Ostrovnyy) [inline submissions/vakulinina/html-movie-seat-booking/index.html]: Why you use tag `b` in this situation?

File: submissions/vakulinina/html-movie-seat-booking/index.html [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+        <p>A deactivated cyborg's revived, but can't remember anything of her past and goes on a quest to find out who
+          she is.</p>
+      </div>
+    </article>
+    <form action="">
+      <fieldset class="date-selector">
+        <legend>Select date</legend>
+        <label class="date-label">
+          <span class="day">Sun</span><span class="date">25</span><span class="month">May</span>
```
Comment (A-Ostrovnyy) [inline submissions/vakulinina/html-movie-seat-booking/index.html]: Use tag `time` this element represents a specific period in time. Also, don't forget about `datetime` attribute.

File: submissions/vakulinina/html-movie-seat-booking/index.html [hunk 5]
```diff
@@ -0,0 +1,204 @@
...
+          <input class="date-input" type="radio" name="date" id="">
```
Comment (A-Ostrovnyy) [inline submissions/vakulinina/html-movie-seat-booking/index.html]: Add attribute `value` to the input with the corresponding value.

File: submissions/vakulinina/html-movie-seat-booking/style.css [hunk 6]
```diff
@@ -0,0 +1,319 @@
...
+  appearance: none;
+}
+
+.date-selector {
+  display: flex;
+  justify-content: space-around;
+  width: 100px;
+}
+
+.date-label {
```
Comment (A-Ostrovnyy) [inline submissions/vakulinina/html-movie-seat-booking/style.css:112]: Add styles on `:hover`, `:active`, and `:focus` states.
Also, add `cursor: pointer` for better identification of interactive elements.

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!
I like your color pallet.
See my comments below.

File: submissions/vakulinina/html-movie-seat-booking/index.html (hunk 3 above)
Comment (vakulinina) [reply submissions/vakulinina/html-movie-seat-booking/index.html]: Well, to make it bold? )) And to highlight it as a kind of sub-header. Should it be `h3` or something?

File: submissions/vakulinina/html-movie-seat-booking/index.html (hunk 1 above)
Comment (vakulinina) [reply submissions/vakulinina/html-movie-seat-booking/index.html:16]: Which tag should I use for the "Buy tickets" in the `header` then?

File: submissions/vakulinina/html-movie-seat-booking/index.html (hunk 3 above)
Comment (A-Ostrovnyy) [reply submissions/vakulinina/html-movie-seat-booking/index.html]: H3 with styles on bold will be good one

File: submissions/vakulinina/html-movie-seat-booking/style.css (hunk 6 above)
Comment (vakulinina) [reply submissions/vakulinina/html-movie-seat-booking/style.css:112]: I already have styles for `:focus` state on the **inputs**. Will add the styles for the rest of the states.

**Comment by vakulinina** [comment]
Thank you a lot for the review ❤️ 
I made changes, except for one of your comments, because I don't see how that would fit the whole structure of the page. Please check.

**Review by A-Ostrovnyy — APPROVED** [review]
Well done!

**Pull request merged**